systems/overview/12-Glossary.md

```diff
@@ -26,14 +26,34 @@ A component is a software unit with a well-defined interface and explicitly spec
 Extensions are software components that administrators can add (or remove) to their MediaWiki application in order to customize how it looks and works.
 Examples: WikibaseClient, UniversalLanguageSelector, OAuth
 
+## [WikibaseClient](https://www.mediawiki.org/wiki/Extension:Wikibase_Client)
+
+WikibaseClient is a [MediaWiki Extension](#mediawiki-extension) that enables the use and display of Entity data from a [Wikibase Repository](#wikibase-repository) via Lua modules or parser functions. Clients can also use centralized language links.
+
+## Wikibase Repository
+
+Wikibase Repository is a structured data repository providing [Entity](#entity) data. It is implemented as a [MediaWiki extension](#mediawiki-extension) called [WikibaseRepo](https://www.mediawiki.org/wiki/Extension:Wikibase_Repository).
+
 ## Wikibase Extension
 
 Wikibase Extensions are MediaWiki extensions that add further functionality to Wikibase. They depend on the Wikibase extension being installed and directly interact with it.
 Examples: WikibaseLexeme, WikibaseMediaInfo, WikibaseImport
 
+## Entity
+
+Entities are the top level concepts of Wikibase's data model. Items and Properties are the core Entity types of Wikibase. Other types can be added through [Wikibase extensions](#wikibase-extension), such as [Lexemes](https://www.mediawiki.org/wiki/Extension:WikibaseLexeme/Data_Model#Lexeme).
+
+## [Item](https://www.wikidata.org/wiki/Help:Items)
+
+In Wikidata, items are used to represent all the things in human knowledge, including topics, concepts, and objects. For example, the "1988 Summer Olympics", "love", "Elvis Presley", and "gorilla" are all items in Wikidata.
```

Comment:
items -> Items?

Reply:
Indeed! Fixed. Now I need to figure out where I copy-pasted that from and fix it there too :D